### DISCUSSION TITLE
pv system sizing issue

### DISCUSSION THREAD

**Discussion by aziz333999** [FZJ-IEK3-VSA/FINE#54] [Q&A] [answered]
hey everybody,
 ...i would like to calculate the optimal Size for a pv  system (one building system ) which can cover  the annual consumption of the building is( 3.3 Gwh/a)
the hourly values of the consumption should be red out of given Excel sheet , the 
 capacity max for the pv system is (1mwp), also the (GHI)values should red out of an excel sheet . , the invest per capacity for ma system is (1000€/Kwp).. also the  commodityCost=0.1914421286 for source 1 (electicity net) ,, after building a code for the above specification i got an optimization result with  capacity   =         ([kW_el]           94.951132) for the pv system (source 2) , which is unrealistic .. any ideas why iam getting such results ?
[sink_1(2).xlsx](https://github.com/FZJ-IEK3-VSA/FINE/files/15145109/sink_1.2.xlsx)
[PV_1.xlsx](https://github.com/FZJ-IEK3-VSA/FINE/files/15145110/PV_1.xlsx)
[code source 1, source 2.txt](https://github.com/FZJ-IEK3-VSA/FINE/files/15145125/code.source.1.source.2.txt)

**Comment by t-gross** [comment, marked as answer]
Thanks for your question. 
I think the problem is how you specified the maximum operation rate values for the PV component. 
In the documentation, you can find the following hint: 
```
        :param operationRateMax: if specified, indicates a maximum operation rate for each location and each time
            step by a positive float. If hasCapacityVariable is set to True, the values are given relative
            to the installed capacities (i.e. a value of 1 indicates a utilization of 100% of the
            capacity). If hasCapacityVariable is set to False, the values are given as absolute values in form
            of the commodityUnit for each time step.
```

You would need to convert your operation values so that the values are between 0 and 1.

**Reply by aziz333999** [reply]
Thanks a lot .. yes .. the issue was how i specified the maximum operation rate values for the PV component.after following the above  steps the optimization results has changed.
Thanks again for the above calrification .
